CHANGELOG.md

```diff
@@ -195,7 +203,13 @@ Other minor additions
   <-≤-connex : Connex _<_ _≤_
   >-≥-connex : Connex _>_ _≥_
 
-  1+n≢0     : suc n ≢ 0
+  1+n≢0  : suc n ≢ 0
+  0≢1+n
+  n<1+n    (as ≤-refl)
+  m*[1+n]
+  2m≢1+2n
+  m>1⇒m*n≢1
+
```

Comment:
Please also add the type signatures of the various results (without mentioning variables
which are implicit).